revised.tex

```diff
@@ -0,0 +1,555 @@
+The solution is to make a project's governance explicit
+so that people know who makes which decisions.
+
+Large, well-established projects that incorporate as non-profits are required to promulgate bylaws,
+such as those for the Python Software Foundation~\cite{psf-bylaws}.
+What smaller projects should do is less well-documented,
+but generally falls under one of three headings~\cite{fogel2005}.
+The first is a ``benevolent dictator'' (often the project founder),
+who the community agrees has final say on important issues.
+This model is common in young or small projects,
+but is brittle,
+and inevitably fosters the emergence of unofficial (and hence unaccountable) de facto leaders
+in specific areas.
+
+The second model formalizes a consensus-building process
+in which the whole community can take part.
+One example is Martha's Rules~\cite{minahan1986} (summarized in the appendix),
+under which anyone can put forward proposals,
+but those proposals are only adopted once it is clear that most people are not strongly opposed.
+The third model is based on elected representation.
+In the Carpentries~\cite{carpentries-bylaws},
+for example,
+the electorate includes anyone who has:
+
+\begin{itemize}
+
+\item
+  completed instructor certification in the preceding year;
+
+\item
+  completed certification in the last two years and taught at least one workshop;
+
+\item
+  been certified for more than two years and has taught at least twice in that time;
+  or
+
+\item
+  made a significant contribution to lesson development, infrastructure, or other activities
+  as determined by the Executive Council.
+
+\end{itemize}
+
+\noindent
+Decisions are then made by the elected,
+though they may decide or be required to take some matters to a referendum vote.
+
+More complex models are possible~\cite{apache-governance},
+but the most important thing is to decide on the rules well in advance of contentious issues emerging,
+since tempers may already be running hot by the time this point is reached.
+
+\rulemajor{Rule 4: Keep knowledge up to date and findable.}
+
+When starting to contribute to a project,
+newcomers must orient themselves in an unfamiliar landscape~\cite{dagenais2010}.
+It is therefore important to make sure that all necessary information is both accessible and findable.
+A single project may use wikis, files in GitHub, shared Google Docs, old tweets or Slack messages, and email archives;
+keeping information about a specific topic in a single place
+and clearly defining the purpose of each communication medium
+saves newcomers from having to navigate multiple unfamiliar data sources to find what they need.
+Doing this makes newcomers more confident and oriented~\cite{steinmacher2016}.
+
+At the same time,
+outdated documentation may lead newcomers to a wrong understanding of the project,
+which is also demotivating.
+While it may be hard to keep material up-to-date,
```

Comment:
```suggestion
While it may be hard to keep material up to date,
```

Reply:
"up to date" is one of those weird modifiers that only gets hyphens when it's before the noun (up-to-date material vs material up to date)

Reply:
:+1: